### PR TITLE
upgrade eslint to 2.0.0

### PR DESCRIPTION
**Pull request by benjycui**
See: [removed rules](http://eslint.org/docs/user-guide/migrating-to-2.0.0#removed-rules)

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -31,7 +31,6 @@
     "no-else-return": 0,
     "no-empty": 2,
     "no-empty-character-class": 2,
-    "no-empty-label": 2,
```
Comment (benjycui) [inline .eslintrc]: [`no-labels`](https://github.com/eggjs/eslint-config-egg/pull/5/files#diff-1dc6ee56b778cd91e0327b52aaeaa1b9R52) 早已存在。

**Comment by popomore** [comment]
+1

**Comment by popomore** [comment]
先不发版本，2.0 还有一些改动

**Comment by benjycui** [comment]
:ok_hand:

**Pull request merged**